lib/errors/DiscordHTTPError.js

```diff
@@ -17,13 +17,19 @@ class DiscordHTTPError extends Error {
             value: response
         });
 
-        this.code = res.statusCode;
+        Object.defineProperty(this, "code", {
+            enumerable: false,
```

Comment:
Object.defineProperty() sets `enumerable` to `false` by default, so it can be removed, but it probably wouldn't matter much

Reply:
considering that people suggested I don't use `Object.defineProperty` if "all I was doing" was `value: thing` it may be best to be more declarative.